coral/src/app/features/topics/request/components/SelectOrNumberInput.tsx

```diff
@@ -0,0 +1,30 @@
+import isNumber from "lodash/isNumber";
+import { NativeSelect, NumberInput } from "src/app/components/Form";
+import { Schema } from "src/app/features/topics/request/schemas/topic-request-form";
+
+type Props = {
+  label: string;
+  name: "topicpartitions" | "replicationfactor";
+  max?: number;
+};
+const SelectOrNumberInput = ({ name, label, max }: Props) => {
```

Comment:
Nitpick: Could this belong in the `Form`, or do you think it is only relevant in the Topic Request scope?

Maybe we can move it once we encounter another situation where it would be handy.

Reply:
I do not think this is yet a "atom" level form component. If we would use same component in a second location, I would do the promotion. At the moment, the form context is pretty hard coded in the `[name](name: "topicpartitions" | "replicationfactor";)` prop :)